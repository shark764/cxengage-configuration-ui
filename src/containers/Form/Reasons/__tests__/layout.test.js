/*
 * Copyright Â© 2015-2018 Serenova, LLC. All rights reserved.
 */

import React from 'react';
import { shallow } from 'enzyme';

import ReasonsForm from '../layout';

describe('<ReasonsForm />', () => {
  it('renders create form', () => {
    const rendered = shallow(
      <ReasonsForm
        name="mockName"
        description="mockDescription"
        externalId={'mockExternalId'}
        userHasUpdatePermission={true}
        userHasSharePermission={false}
        shared={false}
        active={true}
        isSaving={false}
        inherited={false}
        handleSubmit={() => {}}
      />
    );
    expect(rendered).toMatchSnapshot();
  });
  it('renders update form', () => {
    const rendered = shallow(
      <ReasonsForm
        name="mockName"
        description="mockDescription"
        externalId={'mockExternalId'}
        userHasUpdatePermission={true}
        userHasSharePermission={true}
        shared={false}
        active={true}
        id="0000-0000-0000-0000-0000"
        isSaving={false}
        inherited={false}
        handleSubmit={() => {}}
      />
    );
    expect(rendered).toMatchSnapshot();
  });
});
