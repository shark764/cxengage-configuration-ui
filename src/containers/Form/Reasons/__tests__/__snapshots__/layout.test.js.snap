// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReasonsForm /> renders create form 1`] = `
ShallowWrapper {
  "length": 1,
}
`;

exports[`<ReasonsForm /> renders update form 1`] = `
ShallowWrapper {
  "length": 1,
}
`;
